src/RosLib.js

```diff
@@ -8,7 +8,7 @@
  *
  * If you use nodejs, this is the variable you get when you require('roslib')
  */
-var ROSLIB = this.ROSLIB || {
+var ROSLIB = this && this.ROSLIB || {
   REVISION : '1.0.1'
 };
```

Comment:
https://6river.slack.com/archives/C06DJ1Q2SUE/p1711549650572499?thread_ts=1711495726.975329&cid=C06DJ1Q2SUE
```suggestion
const ROSLIB = {
  REVISION : '1.0.1'
};
```